tikv-configuration-file.md

```diff
@@ -248,13 +248,13 @@ Coprocessor 线程池中线程的栈大小，默认值：10，单位：KiB|MiB|G
 ### `scheduler-concurrency`
 
 + scheduler 内置一个内存锁机制，防止同时对一个 key 进行操作。每个 key hash 到不同的槽。
-+ 默认值：2048000
++ 默认值：512 * 1024
 + 最小值：1
 
 ### `scheduler-worker-pool-size`
 
 + scheduler 线程个数，主要负责写入之前的事务一致性检查工作。
-+ 默认值：4
++ 默认值：cpu_num >= 16 { 8 } else { 4 }
```

Comment:
@TomShawn  Is it a preferred format?

Reply:
No. We usually give a definite number, not a format that only technical personnel can understand. @ZhenhanGong